topic-operator/src/main/java/io/strimzi/operator/topic/TopicSerialization.java

```diff
@@ -146,7 +146,7 @@ public static KafkaTopic toTopicResource(Topic topic, Labels labels) {
                     .build();
         }
 
-        KafkaTopic kt = new KafkaTopicBuilder().withApiVersion("v1")
+        KafkaTopic kt = new KafkaTopicBuilder().withApiVersion("kafka.strimzi.io/v1beta1")
```

Comment:
Why is this necessary?

Reply:
I do not know the exact reason, but `fabric8` client needs also an `apiGroup` now.